.github/workflows/build-images-ci-v1.16.yaml

```diff
@@ -208,6 +217,11 @@ jobs:
           df -h
           docker buildx du
 
+      - name: Check build constraints
+        id: check
+        run: |
+          echo build="true" >> $GITHUB_OUTPUT
```

Comment:
Which code is interpreting this `build=true` flag? I couldn't see a match in the same workflow.